recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "1.11.4" %}
+{% set version = "1.10.8" %}
```

Comment:
This is a downgrade not an updated. If you need that version you must submit it to a branch.

Reply:
@ocefpaf : you mean I can submit to any branch except master?

Reply:
I mentioned this before. Older versions that you want to support MUST be on a LTS branch.

First you need to answer my questions:

- why do you need the older version
- are you going to support that older version

and yes, we should put that in a branch that is not `master`.

Reply:
@ocefpaf: I need that version of django for django cookiecutter. I will be maintainer for that version if needed.

Reply:
The question you did not answer yet is: will it be a single release or something for a long term support?

Reply:
cookiecutter needed 1.10.7 version of django, but now requires 1.10.8.   In future, it will require later versions. So, i am not going to need a particular version for a long time.

Reply:
Which one do you wish to add to conda-forge? 1.10.7 or 1.10.8?

Reply:
1.10.8

Reply:
It's been a while since we talked. Do you want me to make a pull request to a non-master branch?  I can't see a non-master branch for 1.10 versions of django.